src/components/chatroom/header/header.chatroom.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import searchIcon from "./mag.png";
+
+type HeaderProps = {
+  onSearchChange: React.ChangeEventHandler<HTMLInputElement>;
+};
+
+function Header({ onSearchChange }: HeaderProps) {
+  let [searchToggle, setSearchToggle] = useState(false);
```

Comment:
```suggestion
  const [searchToggle, setSearchToggle] = useState(false);
```
`state`는 일반적으로 `const`로 선언하는 것 같습니당.
[hooks에서 useState가 const로 선언되는 이유](https://dudghsx.tistory.com/m/18)